src/main/java/com/minecolonies/achievements/AbstractAchievement.java

```diff
@@ -0,0 +1,30 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.stats.Achievement;
+
+/**
+ * Abstraction of the {@link Achievement} class.
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class AbstractAchievement extends Achievement
+{
+    public AbstractAchievement(final String id, final String name, final int offsetX, final int offsetY, final Item itemIcon, final Achievement parent)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)